### DISCUSSION TITLE
Difficulty Installing

### DISCUSSION THREAD

**Discussion by MabryLC** [dolanmiu/docx#1329] [Q&A] [answered]
Hey everyone,

I have been developing an app using docxjs for a while and decided to move it to aspnet so I can use microsoft apis. In the process of updating the app in the new project I ran into some snags with npm and node. I think I changed some versions around or upgraded them. I got the package installed that was causing me trouble but then I tried to install docxjs and here's what happened:

```
mymac@mymacbook clientapp % npm install --save docx   
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@angular-devkit/architect@0.901.15',
npm WARN EBADENGINE   required: { node: '>= 10.13.0', npm: '^6.11.0 || ^7.5.6', yarn: '>= 1.13.0' },
```

It seemed to have installed it but then when I compiled the app for debugging it gave me that message. The next noteworthy thing that happened was some error messages saying that Media.addImage() wasn't part of Media (Property 'addImage' does not exist on type 'typeof Media'). So basically, I had it installed but half of the imports were unusable. I installed npm 6.11.0 since it appears to indicate that that is the only compatible version, but that didn't help anything.

Any idea how I can fix this? Thanks!

**Comment by dolanmiu** [comment, marked as answer]
This is an error related to angular no?

**Reply by MabryLC** [reply]
Yes, sir; it is. It works great now. I thought that error message was talking about the docxjs package. I'm just getting my first software running and making newb mistakes. Thanks so much for working on this and sorry for the newbie questions.
